packages/contentstack-seed/package.json

```diff
@@ -18,8 +16,7 @@
     "tslib": "^1.13.0"
   },
   "devDependencies": {
-    "@oclif/dev-cli": "^1.22.2",
-    "@oclif/plugin-help": "^5.1.12",
+    "oclif": "^3",
```

Comment:
Can we use the latest oclif verion [3.1.2] ?

Reply:
done